en/docs/guides/access-delegation/configure-kerberos-grant.md

```diff
@@ -0,0 +1,142 @@
+# Configure Kerberos Grant
+
+This page guides you through the flow involved in exchanging a Kerberos ticket for an OAuth2 token. To learn more, see [Kerberos grant](insertlink).
+
+----
+
+## Enable kerberos grant
+
+1.  Download the [kerberos grant.jar](../../assets/attachments/kerberos-grant-1.0.0.jar) file.
+
+2.  Copy the JAR into the `<IS_HOME>/repository/components/lib` directory.
+
+3.  Add the following entry to the `<IS_HOME>/repository/conf/deployment.toml` file. 
+
+    ``` toml
+    [oauth.grant_type.kerberos_grant]
+    enable = true
+    ```
+
+-----
+
+{!fragments/register-a-service-provider.md!}
+
+3.  Expand **Inbound Authentication Configuration** section and then **OAuth/OpenID Connect Configuration**. 
+
+4. Click **Configure**.   
+
+5. Select the **Kerberos** from the **Allowed Grant Types** list.
+        
+6. Enter the **Callback Url**.
+
+    !!! tip
+        For more information on `Callback Url` field and other advanced configurations, see [Advanced OpenID Connect Configurations](../../guides/authentication/oauth-app-config-advanced).
+        
+7.  Click **Add**. 
+
+    Note that the **OAuth Client Key** and **Client Secret** are generated. You will need these values later on when sending the request to the authorization endpoint.
```

Comment:
Note that **OAuth Client Key** and **Client Secret** are generated.

Reply:
since client key and secret are nouns, its okay to have the word "the" before it right?